### ISSUE TITLE
Touch Effects: touch controls "gripper" should be a button

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#314]
Element with ID demo-controlsgripper is a div with no label. Should be a button with expand and collapse aria states (it is similar in theory but not visually to the menu buttons in indigo)
